src/views/Votes.vue

```diff
@@ -125,131 +125,153 @@ import AppToolbarCentered from '@/components/AppToolbarCentered'
+        })
+
+        return false
+      }
+      if (store.state.balance < 50) {
+        store.dispatch('snackbar/show', {
+          message: t('votes.no_money')
+        })
+
+        return false
+      }
+
+      return true
+    }
+    const sendVotes = () => {
+      if (!validateVotes()) {
         return
       }
 
-      const upVotes = this.delegates
-        .filter(delegate => delegate.upvoted && !delegate.voted)
-        .map(delegate => `+${delegate.publicKey}`)
-      const downVotes = this.delegates
-        .filter(delegate => delegate.downvoted && delegate.voted)
-        .map(delegate => `-${delegate.publicKey}`)
+      const upVotes = delegates.value
+          .filter(delegate => delegate.upvoted && !delegate.voted)
+          .map(delegate => `+${delegate.publicKey}`)
+      const downVotes = delegates.value
+          .filter(delegate => delegate.downvoted && delegate.voted)
+          .map(delegate => `-${delegate.publicKey}`)
       const allVotes = [...upVotes, ...downVotes]
 
       if (allVotes.length <= 0) {
-        this.$store.dispatch('snackbar/show', {
-          message: this.$t('votes.select_delegates')
+        store.dispatch('snackbar/show', {
+          message: t('votes.select_delegates')
         })
 
         return
       }
 
-      this.waitingForConfirmation = true
-      this.dialog = false
+      waitingForConfirmation.value = true
+      dialog.value = false
 
-      this.$store.dispatch('delegates/voteForDelegates', {
+      store.dispatch('delegates/voteForDelegates', {
         votes: [...upVotes, ...downVotes],
-        address: this.$store.state.address
+        address: store.state.address
       })
 
-      this.$store.dispatch('snackbar/show', {
-        message: this.$t('votes.sent')
+      store.dispatch('snackbar/show', {
+        message: t('votes.sent')
       })
-    },
-    validateVotes () {
-      if (this.upvotedCount + this.downvotedCount > this.voteRequestLimit) {
-        this.$store.dispatch('snackbar/show', {
-          message: this.$t('votes.vote_request_limit', { limit: this.voteRequestLimit })
-        })
-
-        return false
-      }
-
-      if (this.$store.state.balance < 50) {
-        this.$store.dispatch('snackbar/show', {
-          message: this.$t('votes.no_money')
-        })
-
-        return false
-      }
+    }
+    const showConfirmationDialog = () => {
+      dialog.value = true
+    }
 
-      return true
-    },
-    showConfirmationDialog () {
-      this.dialog = true
+    return {
+      search,
+      pagination,
+      expanded,
+      waitingForConfirmation,
+      dialog,
+      t,
+      className,
+      delegates,
+      numOfUpvotes,
+      numOfDownvotes,
+      totalVotes,
+      pages,
+      showPagination,
+      reviewButtonDisabled,
+      getExplorerUrl,
+      upVote,
+      downVote,
+      sendVotes,
+      validateVotes,
+      showConfirmationDialog
```

Comment:
Need to return `VOTE_REQUEST_LIMIT ` because it is used inside the `<template/>`